### PR TITLE
DOC Replace gitter badge with one for slack

### LINKED ISSUE #49143: DOC: replace "join Gitter chat" README badge with link to Slack channel
Let's replace this badge

https://github.com/pandas-dev/pandas/blob/464c10993c6ec76636a1335903f21baf8622c1e0/README.md#L16

with a custom one (e.g. https://gist.github.com/afig/be5ab20c50062dba7cb835e30206659a) which says "Join Slack" and which links to https://pandas.pydata.org/docs/dev/development/community.html?highlight=slack#community-slack

### PR DESCRIPTION
**Pull request by roadswitcher**
- [x] closes #49143
- [ ] [Tests added and passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#writing-tests) if fixing a bug or adding a new feature
- [ ] All [code checks passed](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#pre-commit).
- [ ] Added [type annotations](https://pandas.pydata.org/pandas-docs/dev/development/contributing_codebase.html#type-hints) to new arguments/methods/functions.
- [ ] Added an entry in the latest `doc/source/whatsnew/vX.X.X.rst` file if fixing a bug or adding a new feature.

### CODE REVIEW

**Review by MarcoGorelli — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@
 [![Coverage](https://codecov.io/github/pandas-dev/pandas/coverage.svg?branch=main)](https://codecov.io/gh/pandas-dev/pandas)
 [![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/pandas-dev/pandas/badge)](https://api.securityscorecards.dev/projects/github.com/pandas-dev/pandas)
 [![Downloads](https://static.pepy.tech/personalized-badge/pandas?period=month&units=international_system&left_color=black&right_color=orange&left_text=PyPI%20downloads%20per%20month)](https://pepy.tech/project/pandas)
-[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/pydata/pandas)
+[![slack](https://img.shields.io/badge/join_slack-information-brightgreen.svg?logo=slack)](https://pandas.pydata.org/docs/dev/development/community.html?highlight=slack#community-slack)
```
Comment (datapythonista) [inline README.md:16]: Should `slack` be `Slack` with capital `S`?

Comment (roadswitcher) [reply README.md]: I went with lowercase out of habit/matching most of the other badges.
 
 Shields API does support capitalization here, PR updated.

**Review by mroeschke — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @roadswitcher

**Pull request merged**